ops/charm.py

```diff
@@ -18,32 +18,13 @@
 import enum
 import logging
 import pathlib
-from typing import (
-    TYPE_CHECKING,
-    Any,
-    Dict,
-    List,
-    Literal,
-    Mapping,
-    NoReturn,
-    Optional,
-    TextIO,
-    Tuple,
-    TypedDict,
-    Union,
-    cast,
-)
+from typing import (TYPE_CHECKING, Any, Dict, List, Literal, Mapping, NoReturn,
+                    Optional, TextIO, Tuple, TypedDict, Union, cast)
```

Comment:
It seems annoying to have formatting changes like this, when we're planning to start using Ruff's formatter shortly (which will probably change this back?). Any way to turn off this more aggressive formatting for this PR?

I'd probably even prefer a `noqa` comment on these for now if that helps, rather than reformatting only to use Ruff's formatter in a bit.

Reply:
Ha, I was thinking "might as well just let it format things since we're going to move to ruff's formatter soon", which is basically the opposite :laughing:. I'm fine with a `noqa` for now - I don't think there's any way to have the ruff version of `isort` be less aggressive here.

Reply:
Actually, I might have accidentally done this when doing some auto-fixing. Changing it back doesn't seem to make `check` complain.